airbyte-commons/src/main/java/io/airbyte/commons/json/Jsons.java

```diff
@@ -189,6 +198,25 @@ public static int getIntOrZero(final JsonNode json, final List<String> keys) {
     return optional.map(JsonNode::asInt).orElse(0);
   }
 
+  public static JsonNode flattenConfig(final JsonNode config) {
```

Comment:
shouldn't we check if this jsonNode is an Objectnode before casting implicitly, and return the json node without further processing if it is not an object?